apps/examples/stack_protection/stack_protection_main.c

```diff
@@ -46,29 +83,66 @@ int stack_prot_main(int argc, char *argv[])
 	printf("Stack Overflow Protection testcase!!\n");
 
 	while (1) {
-		printf("\nPress N - Test Stack access within range\n");
-		printf("Press O - Test Stack access in overflow\n");
+		printf("\nPress M - Test Stack protection with MPU\n");
```

Comment:
You should cover this with CONFIG_MPU_STACK_OVERFLOW_PROTECTION as well.
This is the same as register. When you enable REG config, this still is in the list. But we can't test MPU way at that time.

Reply:
done